lib/vanagon/platform/windows.rb

```diff
@@ -107,8 +109,9 @@ def generate_msi_package(project)
         dir_ref = "INSTALLDIR"
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
-        "mkdir -p $(tempdir)/#{project.name}/staging",
-        "gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '$(tempdir)/#{project.name}/staging' --strip-components 1 -xf -",
+        "mkdir -p $(tempdir)/{staging,wix}",
+        "cp -v -r wix/* $(tempdir)/wix",
```

Comment:
You should be able to use `-vr`

Also, you should use `#{@copy}`

Also, I'd rather not have `-v` here because it adds verbose output we don't need. It's fine for debugging purposes. I don't want it in production.